db/primary_migrate/20221109165826_create_irs_attempt_api_log_files.rb

```diff
@@ -0,0 +1,12 @@
+class CreateIrsAttemptApiLogFiles < ActiveRecord::Migration[7.0]
+  def change
+    create_table :irs_attempt_api_log_files do |t|
+      t.string :filename
+      t.string :iv
+      t.text :encrypted_key
+      t.datetime :requested_time, index: true
```

Comment:
I'm not sure `index: true` did what we want? not seeing a corresponding index in `schema.rb`